webui/react/src/components/ResourceChart.tsx

```diff
@@ -29,7 +29,7 @@ const genPlotInfo = (title: string, resources: Resource[]): PlotInfo => {
   const tally = clone(initialTally) as Tally;
 
   resources.forEach(resource => {
-    const state = resource.container ? resource.container.state : ResourceState.Free;
+    const state = resource.container ? resource.container.state : ResourceState.Unspecified;
```

Comment:
This change has visible consequences in the cluster page. But FREE is not defined in the API TS types.

Reply:
`UNSPECIFIED` is a limitation of `protobuf` where enumerated types has to have a fallback value. If it doesn't match anything in terms of resource state, let's say it is considered to be `Free`.

Reply:
I think the right way to handle this is to perform a renaming on the chart level. Lmk if you think that works.

Reply:
I think that is reasonable. Ideally the SDK can send back `FREE` but with the current setup, I agree that probably makes more sense to have the chart interpret Free.

Reply:
That's done on like 41 of this file.